scroll-animations-1/Overview.bs

```diff
@@ -210,6 +210,11 @@ spec: cssom-view-1; type: dfn;
 			then the [=scroll progress timeline=] is [=inactive timeline|inactive=].
 	</dl>
 
+	References to the [=root element=] propagate to the document viewport
+	(which functions as its [=scroll container=]).
+	However in [=paged media=],
+	the [=root element=]’s [=scroll progress timeline=] is [=inactive timeline|inactive=].
```

Comment:
This implies that we have a single scroll progress timeline for anything that references the root (whether explicitly or nearest), but they do have different identities. How about something like:

However in [=paged media=],
[=scroll progress timelines=] referring to the document viewport are [=inactive timeline|inactive=]